### PR TITLE
Add the API to query fhir2datasets

### PR DESCRIPTION
**Pull request by cjumel**
## Description

Add a single-route API to query fhir2dataset, a configuration for formating & for dockerization.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Dockerfile
```
Comment (simonvadee) [inline .dockerignore:1]: what about data/ ? (saw this in gitignore)

**Review by simonvadee — APPROVED** [review]
nice, thanks

**Review by LaRiffle — APPROVED** [review]
🙌

File: .dockerignore (hunk 1 above)
Comment (cjumel) [reply .dockerignore:1]: In this repository there's no need (the .gitignore is just a copy/paste)!

**Pull request merged**